backend/app/models/item.py

```
@@ -0,0 +1,47 @@
+
+
+class Item(ItemBase, BaseModel, table=True):
+    """Item database model."""
+
+    pass
+
+
+class ItemCreate(ItemBase):
+    """Item creation model."""
+
+    pass
```

Ruff (0.12.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)